Packs/GoogleThreatIntelligence/Integrations/GoogleThreatIntelligenceDTMAlerts/README.md

```diff
@@ -0,0 +1,929 @@
+                "topic_matches": [
+                    {
+                        "offsets": [
+                            0,
+                            4
+                        ],
+                        "term": "dummyterm03",
+                        "topic_id": "dummy_topic_01",
+                        "value": "dummyvalue_03"
+                    },
+                    {
+                        "topic_id": "dummay_topic_02",
+                        "value": "message"
+                    }
+                ],
+                "topics": [
+                    {
+                        "entity_locations": [
+                            {
+                                "element_path": "channel.name",
+                                "offsets": [
+                                    0,
+                                    17
+                                ]
+                            }
+                        ],
+                        "extractor": "dtm-ma",
+                        "extractor_version": "1.0.595",
+                        "id": "dummy_topic_01",
+                        "type": "name",
+                        "value": "social"
+                    },
+                    {
+                        "entity_locations": [
+                            {
+                                "element_path": "messenger.name",
+                                "offsets": [
+                                    0,
+                                    8
+                                ]
+                            },
+                            {
+                                "element_path": "channel.messenger.name",
+                                "offsets": [
+                                    0,
+                                    8
+                                ]
+                            }
+                        ],
+                        "extractor": "dtm-ma",
+                        "extractor_version": "1.0.595",
+                        "id": "dummay_topic_02",
+                        "type": "service_name",
+                        "value": "Googlechat"
+                    }],
+                "topics_url": "https://www.virustotal.com/v4/dtm/docs/message/dummy0001/topics",
+                "updated_at": "2025-05-22T07:09:51.257Z"
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### DTM Alerts
+>
+>|Alert ID|Title|Alert Summary|Alert Type|Severity|Status|Monitor ID|Monitor Name|Indicator Score|Created At|Updated At|Tags|
+>|---|---|---|---|---|---|---|---|---|---|---|---|
+>| dummy_0000 | Found topic "google" posted by actor "sevenbump" on channel "social" | This message advertises a service that provides fraudulent Google verification badges for any account. <br> | Message | Low | Read | dummy_monitor_id | Initial Access Broker | 11 | 2025-05-21T11:28:02.646Z | 2025-05-22T07:09:51.257Z | attempt,<br>Test |
```

Comment:
The `<br>` tags in the human-readable output table (lines 475 and 476) are not self-closed. To ensure the documentation renders correctly on MDX-based platforms, please update them to be self-closing (`<br/>`).